frontend/src/components/Template/RightSide/ContributionInfo.spec.js

```diff
@@ -72,7 +72,7 @@ describe('ContributionInfo', () => {
         expect(wrapper.find('p').text()).toBe('contribution.alert.communityNoteList')
       })
 
-      it('has a legend to explain the icons', () => {
+      it.skip('has a legend to explain the icons', () => {
```

Comment:
?

Reply:
Their was no legend anymore but I was not sure if we don't want it later.